src/operator/nn/lrn-inl.h

```diff
@@ -58,8 +58,35 @@ struct LRNParam : public dmlc::Parameter<LRNParam> {
     DMLC_DECLARE_FIELD(nsize)
     .describe("normalization window width in elements.");
   }
+
+  bool operator==(const LRNParam& other) const {
+    return (fabs(this->alpha - other.alpha) < 1e-6 &&
```

Comment:
it’s better to check the nsize first because it’s a far less expensive check than fabs()